rust-escrow/conditional-escrow/src/lib.rs

```diff
@@ -127,7 +142,15 @@ impl ConditionalEscrow {
         let payee = self.get_recipient_account_id();
         let total_funds = self.get_total_funds();
 
-        Promise::new(payee.clone()).transfer(total_funds);
+        // Create Dao
+        ext_dao_escrow::create_dao(
+            "mx".to_string(),           // country_code
+            self.deposits.to_vec(),     // deposits
+            payee.clone(),              // Contract ID
+            self.get_total_funds(),     // funds
```

Comment:
`total_funds` is already declared above.

Reply:
I'm assuming that passing `total_funds` becomes the `env::attached_deposit`?

Reply:
That's correct the `total_founds` becomes the `env::attached_deposit` for the call.